python/ray/serve/schema.py

```diff
@@ -410,6 +412,84 @@ def kubernetes_dict(self, **kwargs) -> Dict:
         return config
 
 
+@PublicAPI(stability="alpha")
+class ServeDeploySchema(BaseModel, extra=Extra.forbid):
+    host: str = Field(
```

Comment:
What's the behavior if users specify the `host` and `port` here as well as the `ServeApplicationSchema`?

Reply:
The host and port fields will be used when deploying through the rest api, and for the new rest api it will simply use the high level `host` and `port` at the `ServeDeploySchema` level and ignore those set at the `ServeApplicationSchema` level. We can maybe do a check at the new rest endpoints and give a warning to users letting them know setting host and port at the `ServeApplicationSchema` level is ineffective?

Reply:
Yeah, can we add the warning?

Reply:
Yup, will do in the upcoming PRs!